### ISSUE TITLE
existence of pytorch code?

### ISSUE THREAD

**Issue by sumukhbhat2701** [layumi/Image-Text-Embedding#18]
Is there any pytorch implementation of the same? Would really appreciate it.

**Comment by layumi** [comment]
Sorry. I do not have enough time to re-train the model. 

But I re-implemented the loss in the competition this year.
You may refer to the code at https://github.com/layumi/NLP-AICity2021

**Comment by sumukhbhat2701** [comment]
I appreciate the reponse. Thanks!
